### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by enumag** [07th-mod/higurashi-patch-compiler#19]
Once again there are quite a few ideas of what we shoud implement with various stages of progress. And since I have some trouble with keeping track of everything here is a list of what I think should be finished for our next major patches.

- [x] LipSync #9 - should be almost finished, needs to be tested but that requires new graphics patches
- [x] Sprite renaming #17 - some minor issues and then recompilation needed
- [x] BGs renaming #18 - need CSVs for renaming then a script to recompile
- [x] Opening poems #20 - needs translation
- [x] Tips upgrade https://github.com/07th-mod/onikakushi/issues/98 - need the last two images and then hack them into the games
- [x] New GUI https://github.com/07th-mod/onikakushi/issues/99 - needs some work and then hack into the game files
- [x] Moving background #24 
- [x] Hinamizawa overview #25
- [x] New text images #27
- [x] Linux case sensitivity https://github.com/07th-mod/onikakushi/issues/96 - need to add a script for linux users to wiki

If you guys have anything to add, let me know.

**Comment by Nozoki17** [comment]
I may have a thing that should interest fan of the anime, I made a patch that replace most of the music with the anime ost, maybe some would like it as an optional patch? I'll upload a video of it later :3

**Comment by enumag** [comment]
@Nozoki17 I love the anime OST so I was actually planning to make exactly that! I even have some early WIP version somewhere. So I really want to get my hands on that patch. 😄  It won't be a part of our main releases but we can certainly list it on the wiki as an optional thing. Can you create a new issue for it?

**Comment by Nozoki17** [comment]
I'll do it as soon as I get home ^^

**Comment by enumag** [comment]
@DoctorDiablo Agreed but this is meant as a rather short-term thread for the next patch so let's keep things like that out of here...

**Comment by enumag** [comment]
@DoctorDiablo Actually there is one more thing we might want to add - there was a discussion about some opening poems on discord. Where are we with those?

**Comment by DoctorDiablo** [comment]
I added the poem to the Onikakushi script.  We need to include the associated images and voice files in the next release.

I don't think we've done anything for the other chapters poems yet, so those need to be translated before we can add them

**Comment by enumag** [comment]
Ok, I'll create a new issue for that.

**Comment by InochiPM** [comment]
I’m currently translating the prologue from sui, it happens before onikakushi and I’m almost positive it happens before every scenario.

**Comment by enumag** [comment]
@InochiPM Yep. As far as I know the story branches at some point after that. I'm not quite sure where to put the prologue though. Maybe into the console arcs release?

**Comment by InochiPM** [comment]
@enumag It's a Sui only thing, because in the updated version of Matsuri, the game just starts without the opening bat scene in onikakushi. In Sui however, it starts with that prologue scene then, it goes straight to the first day of onikakushi. So im going to keep playing through the chapters and such to see if it still branches from that prologue, if anything it would probably go in the beginning of onikakushi so we would have to figure out how to put that in the beginning.

**Comment by enumag** [comment]
@P-Chang or @DoctorDiablo might know more about how exactly the prologue works.

**Comment by P-Chang** [comment]
That is remembrance that Ooishi went on his retirement day
The dialog tells about murder and customs related to Hinamizawa and disasters.

I'm bit against that adding it in opening.

Some of information should be told in Wata but there are no disaster in Wata.
Ooishi is dead in Tata.

I think it is good that add it as tips in console arks.

I may fix choice control, so, I may be able to make tips viewer for console arks(but it is not good as vanilla one.)

**Comment by Grelo** [comment]
I'm against it too these are the unnecessary things PS3 version added.

**Comment by InochiPM** [comment]
@Grelo it was just a suggestion, since it’s labeled as a prologue in the ps3 version. But I thought hat the original versions were going to be modded together with the console arcs at a later point anyway creating that “weird” hybrid?
@P-Chang yeah throwing it into the tips would work, but idk where you would put that in the console arcs? Because it does go before oni.

**Comment by P-Chang** [comment]
@InochiPM 
This scene is not prequel of Onikakushi.
It may be sequel of 1 or 5, or some console arks in timeline.

I don't know where i throw it for now, if there are good point, I may put it.
Basically my plan is this scene is showed in console arks extra menu as default.

Anyway, we never planed combining original arks as much as I remember.

**Comment by enumag** [comment]
> But I thought hat the original versions were going to be modded together with the console arcs at a later point anyway creating that “weird” hybrid?

Nope, that was never the plan. Also MG would not like it because people would not need all their releases. So nope, not doing that.

**Comment by InochiPM** [comment]
@enumag ah then I must have misread somewhere lol sorry.
